backend/src/src-wsimport/src/main/java/org/eclipse/sw360/wsimport/entitytranslation/WsLicenseToSw360LicenseTranslator.java

```diff
@@ -28,6 +28,10 @@ public License apply(WsLicense wsLicense) {
         String licenseShortName = licenseName
                 .replaceAll("[\\s+/]","-");
 
+        if (licenseShortName.endsWith("-")) {
+            licenseShortName = licenseShortName.substring(0, licenseShortName.length()-1) + "+";
+        }
+
```

Comment:
Added this maybe "a bit hacky" way to handle GLP-2.0+, I guess there is no license name that ends with "-"?

Reply:
spdx at least does not forbid to have a `-` as trailing character. And yes, this is way to hacky.

Reply:
removed